### ISSUE TITLE
Note card overflow button not showing in list view

### ISSUE THREAD

**Issue by logickoder** [aritra-tech/Notify#65]
![image](https://github.com/aritra-tech/Notify/assets/42023982/988db7cb-b5df-4b4e-80d4-eb97eb16ef89)

**Comment by aritra-tech** [comment]
Can get it @logickoder, can you please explain in detail

**Comment by logickoder** [comment]
One each item in grid view, there is an overflow icon that allows the user select an action to be done on a note (delete in this case)

![image](https://github.com/aritra-tech/Notify/assets/42023982/3de5b07b-9e6d-4726-94a4-82b6a55027d3)

But that overflow button only shows up in grid view, not in list view

**Comment by aritra-tech** [comment]
Yeah, need to remove the overflow button from the grid view because the delete option is already there in the Edit Screen.
If you want just remove that overflow button from the grid view and create a PR if possible.
And no need to add the overflow button in the List View.
I hope you got it @logickoder

**Comment by logickoder** [comment]
Okay, noted

**Comment by aritra-tech** [comment]
Should I assign it to you?

**Comment by logickoder** [comment]
Yeah, sure

**Comment by aritra-tech** [comment]
When starting with this issue please update the `develop` branch and create a new branch from `develop` with the proper name which can be understandable to me.
I hope you got it @logickoder

**Comment by logickoder** [comment]
I don't understand what you mean by update the develop branch

**Comment by aritra-tech** [comment]
Pull the develop branch once.
